janusgraph-dist/src/test/java/org/janusgraph/pkgtest/JanusGraphAssemblyBaseIT.java

```diff
@@ -128,6 +128,9 @@ protected static void unzip(String dir, String zipFile) throws IOException, Inte
 
     protected static void command(File dir, String... command) throws IOException, InterruptedException {
         ProcessBuilder pb = new ProcessBuilder(command);
+
+        Map<String, String> env = pb.environment();
```

Comment:
Why was this change necessary? Is that related to the changes in `janusgraph-server.sh`?

Reply:
Yeah, it now reads the value from options. It would set the memory to 4G and github actions doesn't have enough memory/